packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -1015,7 +978,7 @@ export class UnfurlService extends BaseService {
 
                     Sentry.captureException(e);
                     hasError = true;
-                    span.recordException(e);
+                    span.addEvent(e);
```

Comment:
Do you have docs for this change? I couldn't find a reference to `addEvent` for recording errors

Reply:
https://github.com/getsentry/sentry-javascript/blob/1201eb2d78425709293ee0aac180cbcc1f4e99d2/packages/types/src/span.ts#L247 saw this @owlas Perhaps we can even remove this because we already capture the exception on the line above, but I would like to attach this event to the `span`